pom.xml

```diff
@@ -80,6 +80,20 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.build.resourceEncoding>UTF-8</project.build.resourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+        <GIT_COMMIT>unknown</GIT_COMMIT>
```

Comment:
Add a comment above this line to break things up, like `<!-- docker build support -->`